### PR TITLE
dev-util/flawfinder: bump to 2.0.7

### PR DESCRIPTION
**Pull request by oxr463**
Bugs: https://bugs.gentoo.org/673858

### CODE REVIEW

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 )
```
Comment (hydrapolic) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: Please consider adding python 3_5 and maybe even 3_7.

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+inherit distutils-r1
+
+DESCRIPTION="Examines C/C++ source code for security flaws"
+HOMEPAGE="https://www.dwheeler.com/flawfinder/"
+SRC_URI="https://www.dwheeler.com/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
```
Comment (hydrapolic) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: Please sort.

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}"
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	local DOCS=( announcement ChangeLog README.md ${PN}.pdf )
+	doman ${PN}.1 || die 'doman failed.'
```
Comment (hydrapolic) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: I thought doman() calls die on it's own if it fails (maybe i'm wrong, haven't tested).

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild (hunk 2 above)
Comment (oxr463) [reply dev-util/flawfinder/flawfinder-2.0.7.ebuild]:
```ebuild
KEYWORDS="~amd64 ~amd64-linux ~mips ~ppc ~ppc-macos ~sparc ~sparc-solaris ~x86 ~x86-linux ~x86-solaris"
```

Repoman still says these keywords are unsorted? I tried alphabetical order, maybe I missed something?

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild (hunk 1 above)
Comment (oxr463) [reply dev-util/flawfinder/flawfinder-2.0.7.ebuild]:
```diff
-PYTHON_COMPAT=( python2_7 python3_6 )
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild (hunk 2 above)
Comment (oxr463) [reply dev-util/flawfinder/flawfinder-2.0.7.ebuild]:
```terminal
  KEYWORDS.unsorted             1
   dev-util/flawfinder/flawfinder-2.0.7.ebuild contains unsorted keywords
```

Comment (Zlogene) [reply dev-util/flawfinder/flawfinder-2.0.7.ebuild]: please use ekeyword from the app-portage/gentoolkit

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+src_prepare() {
+	rm -f "${PN}.1.gz" || die 'rm failed.'
+	sed -i "s/${PN}.1.gz/${PN}.1/g" setup.py || die 'sed failed.'
+
+	default
+}
+
+src_test() {
+	make test
```
Comment (Zlogene) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: I think you may want emake

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Examines C/C++ source code for security flaws"
+HOMEPAGE="https://www.dwheeler.com/flawfinder/"
+SRC_URI="https://www.dwheeler.com/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+IUSE="doc test"
```
Comment (Zlogene) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: where are these flags used?

File: dev-util/flawfinder/metadata.xml [hunk 6]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>ramage.lucas@protonmail.com</email>
+		<description>Lucas Ramage</description>
+	</maintainer>
 </pkgmetadata>
```
Comment (juippis) [inline dev-util/flawfinder/metadata.xml]: You'll also need to add a gentoo developer or a project to commit for you.

File: dev-util/flawfinder/flawfinder-2.0.7.ebuild [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	rm -f "${PN}.1.gz" || die 'rm failed.'
```
Comment (mgorny) [inline dev-util/flawfinder/flawfinder-2.0.7.ebuild]: Why are you removing it? Like, the following sed should be sufficient.

**Comment by oxr463** [comment]
@mgorny, please check my fixes.

**Pull request closed without merging**